code/modules/research/techweb/all_nodes.dm

```diff
@@ -431,10 +431,12 @@
 		"limbdesign_ethereal",
 		"limbdesign_felinid",
 		"limbdesign_lizard",
-		"limbdesign_plasmaman",//honk start - Adding the Skaven and kitsune Designs to the Research
+		"limbdesign_plasmaman",
 		"limbdesign_skaven",
-		"limbdesign_kitsune",//honk end
-	)
+		"limbdesign_kitsune",
+		"limbdesign_plasmaman",
```

Comment:
remove duplicated lines, make sure honk comment stays in. since our modification is only the skaven line can do a single comment at the end of the line